scripts/git-setup.sh

```diff
@@ -56,14 +56,14 @@ install_format() {
   git config --local --get alias.format >/dev/null
   if [ $? -ne 0 ] || [ "$force" = TRUE ]; then
     echo 'adding alias for `git format`'
-    git config --local alias.format "!f() { git diff -U0 --no-color \$@ | ${format_call} -i -p1; }; f"
+    git config --local alias.format "!f() { git diff -U0 --no-color \$@ -- \"*.cpp\" \"*.hpp\" | ${format_call} -i -p1; }; f"
```

Comment:
Same as above in the CI, this is incomplete.

Reply:
Same here @tobim